src/cclib/parser/orcaparser.py

```diff
@@ -163,6 +163,51 @@ def extract(self, inputfile, line):
 
             self._append_scfvalues_scftargets(inputfile, line)
 
+        """
+-------------------------------------------------------------------------------
+                          DFT DISPERSION CORRECTION
+
+                              DFTD3 V3.1  Rev 1
```

Comment:
Is the dispersion version parsed somewhere else? Shouldn't this be done?

Reply:
Yes, but I'm hesitant to just stick it in `metadata`, because I think there is a better solution: If the standalone DFTD3 program produces output that is formatted the same as the above, we could use this as a test case for instantiating a parser inside a parser. It is probably worth having a DFTD3 parser.